config/db.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "donttry123",
+    "database": "dami-dev",
+    "host": "0.0.0.0",
+    "port": 5432,
+    "dialect": "postgres"
+  },
+  "production": {
```

Comment:
Storing configurations for production environment directly in code is not secure